src/main/java/seedu/address/model/eatery/Eatery.java

```diff
@@ -15,26 +15,46 @@
 
     // Identity fields
     private final Name name;
+    private final boolean isOpen;
     // Data fields
     private final Address address;
     private final Category category;
     private Set<Tag> tags = new HashSet<>();
 
     /**
      * Every field must be present and not null.
+     * Used when adding a eatery for the first time
      */
     public Eatery(Name name, Address address, Category category, Set<Tag> tags) {
-        requireAllNonNull(name, address, category, tags);
+        requireAllNonNull(name, address, tags);
```

Comment:
why did you delete category?

Reply:
oops my bad, might have accidentally deleted that. But the user cant create an eatery without category at the start?

Reply:
yeah they can't hahaha like if u refer to our userguide, only tags are optional? so i think u can remove requireAllNonNull for tags? idk